### PR TITLE
Replace Katello PR test for Foreman with a pipeline based one

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: theforeman.org/scripts/test/test_katello.sh [hunk 1]
```diff
@@ -1,79 +0,0 @@
-#!/bin/bash -ex
```
Comment (ekohl) [inline theforeman.org/scripts/test/test_katello.sh:1]: This file is still used - probably in the source build step for the nightly package.

File: theforeman.org/pipelines/test/foreman/vars.groovy [hunk 2]
```diff
@@ -1,15 +1,19 @@
 def foreman_integration_versions = [
     'develop': [
-        'ruby': ['2.7']
+        'ruby': ['2.7'],
+        'katello': ['master']
```
Comment (ekohl) [inline theforeman.org/pipelines/test/foreman/vars.groovy:4]: We have the same table elsewhere, but transposed. I wonder if we can reuse this data somehow. Perhaps just store it in the same file so you only need to update a single file?
https://github.com/theforeman/jenkins-jobs/blob/036271a98c4952aa7b620d33081762a617d01901/theforeman.org/pipelines/test/testKatello.groovy#L1-L27

File: theforeman.org/scripts/test/test_katello.sh (hunk 1 above)
Comment (ehelms) [reply theforeman.org/scripts/test/test_katello.sh:1]: It's used by plugin testing for foreman-virt-who-configure, I meant to not trash it yet.

File: theforeman.org/pipelines/test/foreman/vars.groovy (hunk 2 above)
Comment (ehelms) [reply theforeman.org/pipelines/test/foreman/vars.groovy:4]: That crossed my mind while writing this up. I was thinking to introduce this first and then come back with a round of re-factor to look at the landscape of where this info is stored to centralize it in a way that makes sense.

**Review by ekohl — APPROVED** [review]

**Pull request merged**